README.md

```diff
@@ -15,176 +19,156 @@
+
+3. Add data by importing the JSON file from the `database.mongo` folder: `{DatabaseName}.{CollectionName}.json`
 
-3.  Add data by importing json file:
-    From 'database.mongo' folder -> `{DatabaseName}.{CollectionName}.json`
 
 The database contains a `faceEntries` collection with the following schema:
 
 - `id`: A unique identifier for the face entry.
-- `Employeecode`: A unique  employee ID associated with the image.
+- `Employeecode`: A unique employee ID associated with the image.
 - `Name`: The name of the person in the image.
 - `gender`: The gender of the person.
-- `Department`: The department of the person
+- `Department`: The department of the person.
 - `time`: The time the face entry was created.
 - `embeddings`: The embeddings of the face image.
-- `Image`: Base64 encoded image file.
+- `Images`: Base64 encoded image file.
 
 ## Function Flow
 
-1. `create_new_faceEntry()`: This function receives a POST request with an image and metadata. It extracts the face from the image, calculates the embeddings, and stores the data in the database.
-
-2. `Data()`: This function sent a GET request  to `/data` endpoint of FastAPI app to get the list of Face Entries from MongoDB.
-
-3. `update()` :This function is used to update the details of the face entry in the database.
-
-4. `read()` : This function sent a GET request with specific Employeecode to read the information related to that particular Employeecode.
-
-5. `delete()` : This function is used to delete the specific Employee Data.
-
-# Face Recognition Project
-
-This project uses Flask, FastAPI,DeepFace and MongoDB to create a Face recognition system. It allows users to register face with associated metadata, update their information and also can delete their data.
-
-### Get started
-
-These instructions will get you a copy of the project up and running on your local machine for development.
-
-### Prerequisites
-
-This project requires Python 3.7 or later.
-
-### Installing
-
-1. Clone this repository to your local system using link
-
-    ```bash
-    git clone "https://dev.azure.com/tmspl/FaceRec/_git/FaceRec-Employee-Enrollment.python"
-    ```
-
-2. Navigate to the project directory:
-
-    ```bash
-    cd FaceRec
-    ```
-
-3. Install the required packages:
-
-    ```bash
-    pip install -r requirements.txt
-    ```
-
-
-### Running the Server
-To start FLask and FastAPI, run the given command:
-```bash
-python main.py
+1. `create_new_faceEntry()`: Receives a POST request with an image and metadata. It extracts the face, calculates the embeddings, and stores the data in the database.
+2. `Data()`: Sends a GET request to the `/data` endpoint of the FastAPI app to get the list of Face Entries from MongoDB.
+3. `update()`: Updates the details of the face entry in the database.
+4. `read()`: Sends a GET request with a specific `Employeecode` to read the related information.
+5. `delete()`: Deletes the specific employee data.
+
+## Sequence Diagram
```

Comment:
From here the sequence diagram starts.

Reply:
This is the plant uml for diagram u may need to compile and render it and then put it here